fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/OrderByAdapter.java

```diff
@@ -13,13 +13,22 @@
     // The builder from which we were created
     private final Select select;
 
+    // The order by clause under construction
+    private final OrderByClause orderByClause;
+
     /**
      * Protected constructor for modeling the GROUP BY part of a SELECT statement.
-     * 
+     *
      * @param select
      */
-    protected OrderByAdapter(Select select) {
+    protected OrderByAdapter(Select select, OrderByClause ob) {
         this.select = select;
+        this.orderByClause = ob;
+    }
+
+    public OrderByAdapter add(String...strings) {
+        this.orderByClause.add(strings);
+        return this;
```

Comment:
There are zero references to this method... is this actually used?

Reply:
It's there to make the fluent pattern complete.